docs/src/components/productBaseUI/BaseUIComponents.tsx

```diff
@@ -182,15 +180,15 @@ export default function BaseUIComponents() {
               >
                 <HighlightedCode
                   copyButtonHidden
-                  component={MarkdownElement}
+                  language="jsx"
+                  plainStyles
```

Comment:
```suggestion
                  plain
```
No need for styles as a suffix.

Reply:
Uhm, I feel like just "plain" is not very informative, you'd probably need to open up the component to see what it does exactly. We could go with a bit longer name but more descriptive, such as "noBlockStyle" or similar.